src/user/mod.rs

```diff
@@ -419,54 +391,3 @@ pub fn updates(req: &mut Request) -> CargoResult<Response> {
         meta: Meta { more: more },
     }))
 }
-
-#[cfg(test)]
```

Comment:
Does this PR make these tests invalid? It doesn't seem like the behavior they're testing has changed, and I'm not sure where these tests moved to